src/Components/CreateImageWizard/steps/Registration/ActivationKeysList.tsx

```diff
@@ -136,7 +95,7 @@ const ManageKeysButton = () => {
       isInline
       href={isProd() ? ACTIVATION_KEYS_PROD_URL : ACTIVATION_KEYS_STAGE_URL}
     >
-      Activation keys page
+      Customer portal
```

Comment:
I don't know if this comes from mocks, but it's wrong.
Customer portal is where AKs were managed in the past, before we've got Activation keys page.
If we'd want to link  to customer portal, we should link https://access.redhat.com/
But if I'm correct, AKs can't even be managed there anymore.

Reply:
Oh, good to know. Yeah, it's from the mocks, but this part might not have been updated then. I'll double check and update the copy. Thanks for the heads up!